images/iscsi-demo-target-tgtd/Dockerfile

```diff
@@ -26,18 +26,28 @@ RUN dnf -y update && dnf -y install scsi-target-utils bzip2 qemu-img \
 
 RUN mkdir -p /volume
 
-# Add alpine image
+# LUNs:
+# 0. is an internal lun
+# 1. Empty image - for your data
+
+# 2. Add alpine image
 RUN curl \
       https://nl.alpinelinux.org/alpine/v3.5/releases/x86_64/alpine-virt-3.5.1-x86_64.iso \
       > /volume/alpine.iso
 
-# Add cirros
+# 3. Add cirros
 RUN curl \
       http://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img \
       > /volume/cirros.img \
       && qemu-img convert -O raw /volume/cirros.img /volume/cirros.rawa \
       && rm volume/cirros.img
 
+# 4. Add fedora
+# needs cloud init for login
+RUN curl -L \
```

Comment:
I am not super happy with adding this huge blob to our container builds.

Reply:
+1, OTOH cirrus/alpine are slowly proving to be insufficient for virtualization testing needs

Reply:
@rmohr Yes, I share your perspective to a degreee, that is why I didn't push for it earlier, because of size.
But it looks like the missing guest support is impacting our CI, thus we need a guest with sufficient drivers.

But there is an email thread on that topic now - side note: OSP started with cirros, but added Fedora for the same reason (guest support).

Reply:
we should consider using a registry disk for fedora. It lets us use qcow2 instead of requiring download and expansion of a raw disk into the container image.

Reply:
also, why not FedoraAtomic instead of fedora?

Reply:
FedoraAtomic doesn't have qemu, IIRC

Reply:
Also, with the new pod networking, we won't be able to use iscsi directly. It would be better to have it as a local image.